tests/utils/dockercli/containers.go

```diff
@@ -86,3 +86,24 @@ func GetVolumeProperties(volumeName, hostName string) (string, error) {
 	return ssh.InvokeCommand(hostName, cmd)
 }
 
+// WriteToContainer write data to a file in the running container
+func WriteToContainer(ip, containerName, volPath, fileName, data string) (string, error) {
```

Comment:
We can't use that util here. That util writes data to a volume by running a **new** container. But this util writes data to an **existing** container scheduled by swarm cluster.